sympy/stats/rv.py

```diff
@@ -514,6 +515,33 @@ def conditional_space(self, condition, normalize=True, **kwargs):
 
         return space(domain, density)
 
+class DependentPSpace(PSpace):
+
+    @staticmethod
+    def _create_MutlivariateNormalDistribution(syms, assumps):
+        from sympy.stats.symbolic_probability import Covariance, Variance
+        from sympy.stats.crv_types import Normal
+        assumps = assumps & And.fromiter([Eq(Variance(sym), sym.pspace.distribution.args[1])
+                                            for sym in syms])
+        xrep_dict = {eq.lhs: eq.rhs for eq in assumps.args}
+        mu = [sym.pspace.distribution.args[0] for sym in syms]
+        sigma = ImmutableMatrix(
+                [[Covariance(s1, s2).expand() for s2 in syms] for s1 in syms]
+                ).xreplace(xrep_dict)
+        return Normal('_', mu, sigma).pspace.distribution
+
+
+    @staticmethod
+    def compute_density(expr, assumps):
```

Comment:
What kind of assumptions of dependence does this support, like Covariance or any other metric? What's popular in the literature?

Reply:
Currently, I have added the Covariance